### ISSUE TITLE
Requiring moneysafe/ledger is broken

### ISSUE THREAD

**Issue by nexdrew** [ericelliott/moneysafe#33]
In version 2.2.0, you cannot load the ledger module as intended via `'moneysafe/ledger'` since it points to the v1 directory path of `'../source/ledger'`.

Attempting to require it like so:

```js
const { $$, subtractPercent, addPercent } = require('moneysafe/ledger')
```

Throws an error:

```
Thrown:
Error: Cannot find module '../source/ledger'
Require stack:
- /private/tmp/fakeproject/node_modules/moneysafe/ledger/index.js
- <repl>
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:961:17)
    at Function.Module._load (internal/modules/cjs/loader.js:854:27)
    at Module.require (internal/modules/cjs/loader.js:1023:19)
    at require (internal/modules/cjs/helpers.js:72:18) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/private/tmp/fakeproject/node_modules/moneysafe/ledger/index.js',
    '<repl>'
  ]
}
```

Until this is fixed, the only way to use the ledger module is to reference it this way:

```js
const { $$, subtractPercent, addPercent } = require('moneysafe/src/ledger')
```

Looks like this affects versions 2.0.0 and 2.1.0 as well.

**Comment by ericelliott** [comment]
Open to a PR

**Comment by nexdrew** [comment]
See #34

**Comment by ericelliott** [comment]
Fixed in moneysafe@2.2.1

**Comment by ericelliott** [comment]
Thanks!
